tuna/miopen/db/miopen_tables.py

```diff
@@ -307,6 +309,22 @@ class FusionConfigTags(BASE, ConfigTagMixin):
   config = Column(Integer, ForeignKey("fusion_config.id"), nullable=False)
 
 
+class MiopenJob():
```

Comment:
needs to inherit from BASE and JoxMixin otherwise you are breaking all the tables